### ISSUE TITLE
Phate using harmony reduction

### ISSUE THREAD

**Issue by wiktormiko** [KrishnaswamyLab/PHATE#149]
Hi, I was wondering if I could use Phate with harmony corrected PC's. Is there a way to skip calculating PCA while running PHATE and use pre-computed Harmony embeddings instead?

**Comment by nishuang83** [comment]
Yes, you can set n_pca to None and input your precomputed embeddings. 
```
phate_op = phate.PHATE(n_pca=None)
phate_op.fit_transform(your_embeddings)
```

**Comment by SSSJe** [comment]
By the way, I was wondering if I could use PHATE with PCA corrected PC's (from seurat). Is there a way to skip calculating PCA while running PHATE and use pre-computed PCA embeddings instead?

**Comment by MattScicluna** [comment]
Yes there is! Pass PCA transformed vars as your input and then set n_pca=None. Please see: https://phate.readthedocs.io/en/stable/api.html
